src/sentry/api/bases/release.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.models import (
+    GroupLink,
+    GroupResolution,
+    Release,
+    ReleaseCommit,
+)
+
+
+class IssuesResolvedInReleaseMixin(object):
```

Comment:
Does this need to be a mixin or an instance method? Seems like something that could be a module function avoiding inheritance entirely.

Reply:
Seems reasonable, any thoughts on where we should put it? I don't have a great feel for where these sorts of functions could go, and don't really want to just keep dumping things into utils.

Reply:
Could be in this module.

Reply:
Discussed on slack, decided to put in `api/helpers/releases.py`